### PR TITLE
Home Task 3

### PR DESCRIPTION
**Pull request by Rehaeel**
I added a `name` to the login page because there is no way to guess the user name.

I also added a `name` to localStorage because I can't fetch name from `store`  on refresh or second visit if a token is available;

### CODE REVIEW

File: src/components/CourseInfo/CourseInfo.jsx [hunk 1]
```diff
@@ -1,56 +1,46 @@
...
-	useEffect(() => {
-		const fetchedCourse = mockedCoursesList.filter(
-			(course) => course.id === courseId
-		);
-		setCourse(...fetchedCourse);
-		setIsFetched(true);
-	}, [courseId]);
+	const authors = useSelector(selectAuthors);
+	const courses = useSelector(selectCourses);
+	const [filteredCourse] = courses.filter((course) => course.id === courseId);
```
Comment (jbbch) [inline src/components/CourseInfo/CourseInfo.jsx:14]: use find instead of filter if you need one item from array

File: src/store/services.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+import { BACKEND_URL } from '../constants';
+
+export const fetchedCourses = axios.get(`${BACKEND_URL}/courses/all`);
```
Comment (jbbch) [inline src/store/services.js:4]: I would say that's a bad naming, constant is not a courses list, right? It's just a promise that needs to be called, so it's function, hence fetchCourses is a better name here

File: src/components/CourseInfo/CourseInfo.jsx (hunk 1 above)
Comment (Rehaeel) [reply src/components/CourseInfo/CourseInfo.jsx]: good point.
changed

File: src/store/services.js (hunk 2 above)
Comment (Rehaeel) [reply src/store/services.js:4]: Understand, seems clear. Could it be: `queryCoursesResponse`?

**Pull request merged**